.flowconfig

```diff
@@ -6,6 +6,7 @@
 <PROJECT_ROOT>/.nyc_output/.*
 .*/tmp/**/.*
 .*/.parcel-cache/.*
+.*/dist/.*
```

Comment:
won't this capture any dist files in `node_modules`? There are a couple of situations where we rely on the published flow sources for their typings, rather than using definition files.

Reply:
hmm yeah I guess so. This was because I was tired of flow going crazy when I ran the tests and it created a lot of files.